service/src/test/java/org/exoplatform/lecko/service/TestUserActivityWithCommentAComment.java

```diff
@@ -92,6 +85,9 @@ public void testBuildUserExportWithCommentAComment() throws Exception {
     String ls = System.getProperty("line.separator");
     String[] lines = fileContent.split(ls);
 
+    // Assert that the file contain 9 lines.
+    assertEquals(9, lines.length);
```

Comment:
I think we should update the test TestSpaceActivityWithCommentAComment with the same assert.